### ISSUE TITLE
Different normalization processing during the validation process and training process

### ISSUE THREAD

**Issue by jackhu-bme** [ibrahimethemhamamci/CT-CLIP#36]
The normalization range during training and inference is different. During training, [-1000, 1000] -> [-1, 1]. while during inference, [-1000, 200] -> [-1, 1]. Why? This might cause performance drop severely!

Based on:
The release code of scripts/data.py, in nii_img_to_tensor function of CTReportDataset, the range is [-1000, 1000].
However, the release code of scripts/data_inference.py, in nii_img_to_tensor function of CTReportDatasetInfer, the range is [-1000, 200].

Please tell us why, or we are hard to follow this setting. We believe consistent pre-processing between train and infer is very very important. Seem like a little but serious mistake here!

**Comment by sezginerr** [comment]
Hi @jackhu-bme,

We have decided to change the clipping range from -1000 to 1000 to include other parts of the CT, including bones, in the second iteration. This was necessary for the subsequent CT-CHAT training (bones etc. are also evaluated in the reports.) Apparently, I did not update the inference dataloader script when I cleaned the code for the second iteration. I will fix it, but inference is also done between -1000 and 1000. Thanks for pointing this out!

**Comment by sezginerr** [comment]
Hi @jackhu-bme, we have updated the preprint recently. Please see: https://arxiv.org/pdf/2403.17834 for the updated preprint and https://github.com/ibrahimethemhamamci/CT-CHAT for GitHub repo.

**Comment by Farazsh** [comment]
Hello @sezginerr, 
Did you already upload the inference dataset script, i.e. data_inference_nii.py? I couldn't find it in the scripts folder.
